apps/frontend/src/store/services/user.api.ts

```diff
@@ -7,7 +8,41 @@ const userApi = serverApi.injectEndpoints({
     Login: builder.mutation<LoggedUser, Login>({
       query: (body) => ({ url: 'login', method: 'POST', body }),
     }),
+    UpdateUser: builder.mutation<number, UpdateUser>({
+      query: (body) => ({ url: `user/${body.id}`, method: 'PATCH', body }),
+      invalidatesTags: ['toasts', 'user'],
+    }),
+    Signup: builder.mutation<LoggedUser, Login>({
+      query: (body) => ({
+        url: 'signup',
+        method: 'POST',
+        body,
+      }),
+    }),
+    GetUsers: builder.query<
+      { username: string; id: string; isAdmin: boolean }[],
+      void
+    >({
+      query: () => ({
+        url: 'users',
```

Comment:
```suggestion
        url: 'user',
```
It's better to be consistent - choose either api/user OR api/users